prow/cluster/BUILD.bazel

```diff
@@ -11,6 +11,7 @@ release(
     component("deck", "service", "deployment"),
     component("gce-ssd-retain", "storageclass"),
     component("ghproxy", "service", "deployment"),
+    component("grandmatriarch", MULTI_KIND, cluster=BUILD_CLUSTER),
```

Comment:
do we want to add it to our prow deployment?

Reply:
That is what this does?

Reply:
(I mean we want to add it to the gerrit deployment, not k8s one? or given nobody is using the token so this is fine I guess)

Reply:
I think this is okay in the k8s cluster?